worker/contrib/windows/buildbot_service.py

```diff
@@ -104,7 +104,7 @@
 class BBService(win32serviceutil.ServiceFramework):
     _svc_name_ = 'BuildBot'
     _svc_display_name_ = _svc_name_
-    _svc_description_ = 'Manages local buildbot slaves and masters - ' \
+    _svc_description_ = 'Manages local buildbot workers and masters - ' \
                         'see http://buildbot.sourceforge.net'
```

Comment:
FWIW I'm not *terribly* worried about anything in `contrib/`.  It's fine to fix things as you see them, but don't put too much effort into it.